src/Field/NominatimSearch/NominatimSearch.spec.jsx

```diff
@@ -79,11 +90,12 @@ describe('<NominatimSearch />', () => {
   describe('#onFetchError', () => {
     it('sets the response as state.dataSource', () => {
       const wrapper = TestUtil.mountComponent(NominatimSearch);
-      const loggerSpy = sinon.spy(Logger, 'error');
+      const loggerSpy = jest.spyOn(Logger, 'error');
       wrapper.instance().onFetchError('Peter');
-      expect(loggerSpy.calledOnce).toBeTruthy();
-      expect(loggerSpy.calledWith('Error while requesting Nominatim: Peter')).toBeTruthy();
-      loggerSpy.restore();
+      expect(loggerSpy).toHaveBeenCalled();
+      expect(loggerSpy.mock.calls[0][0]).toBe('Error while requesting Nominatim: Peter');
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith